### ISSUE TITLE
Partial differential symbol ∂ (del, curly d) closer to 6 and to ð

### ISSUE THREAD

**Issue by kauesena** [be5invis/Iosevka#862]
- [X] The requested variant shape does not go too far away from Iosevka's design.
- [X] The requested variant does not conflict with any characters in Unicode that Iosevka currently supports.
- [X] Some other monospace/programming fonts supported the requested variant. Provide images below.

------
It would be nice to have a variant of `∂` closer to the second form of `6` mirrored or to the character `ð` without the bar. 
![image](https://user-images.githubusercontent.com/30255100/106396188-1f605680-63e5-11eb-9225-efc4326667cc.png)
![image](https://user-images.githubusercontent.com/30255100/106396215-4fa7f500-63e5-11eb-8028-81d1f5e47fa4.png)

**Comment by be5invis** [comment]
This symbol **should** have a significant hook at the top, as far as I know, all math fonts are implemented with a significant hook.
Mark as Needs Justification.

**Comment by kauesena** [comment]
I do not really agree with you about the _should_. However, I certainly agree that the most common form we see that character in mathematical typesetting is with a very significant hook due to the foundational importance in digital mathematical typesetting and widespread use of Computer Modern Roman (which realizes the chracter `∂` in such a way). 

The needed justification would be other fonts realizing this character with less significant hooks?  If so, I believe I can provide it. Have a look at these samples I gathered from fonts I have installed in my computer.
![image](https://user-images.githubusercontent.com/30255100/106407582-2dc96500-641b-11eb-889c-5809ade82d18.png)
In this sample are some fonts which are used with LaTeX for math: Latin Modern Math (successor of Computer Modern), Libertinus Math, Cambria Math, DejaVu Math, Erewhon Math, STIX Two Math, Tex Gyre Bonum Math, Minion Math, Utopia Std and Fira Math.

In the next image there are some programming fonts: 
![image](https://user-images.githubusercontent.com/30255100/106407705-7e40c280-641b-11eb-8dbe-82ede58319a9.png)
In it are: Iosevka mono, Liberation mono, Menlo, IBM Plex  Mono, Jet Brains Mono, Source Code Pro and Victor Mono.
